x-pack/plugins/observability_solution/synthetics/public/apps/synthetics/components/monitors_page/overview/overview/metric_item.tsx

```diff
@@ -136,10 +140,51 @@ export const MetricItem = ({
                 {
                   title: monitor.name,
                   subtitle: locationName,
-                  value: stats.medianDuration,
+                  value: trendData?.median ?? 0,
                   trendShape: MetricTrendShape.Area,
-                  trend: data,
-                  extra: <MetricItemExtra stats={stats} />,
+                  trend: trendData?.data ?? [],
+                  extra: trendData ? (
```

Comment:
i don't understand why not sue Metric item Extra?

Reply:
That's the plan, something else that got moved around during some initial coding that I haven't cleaned up yet 😄.